src/config.ts

```diff
@@ -21,20 +21,26 @@ try {
 
 export const yamlConfig = _.merge(defaultConfig, customConfig)
 
-export class TransactionLimits implements ITransactionLimits {
-  readonly config
-  readonly level
-
-  constructor({ level }) {
-    this.config = yamlConfig.limits
-    this.level = level
+export const generateLimitConstants = (
+  limitsConfig = yamlConfig.limits,
+): LimitConstants => ({
+  oldEnoughForWithdrawalHours: limitsConfig.oldEnoughForWithdrawal / MS_IN_HOUR,
+  oldEnoughForWithdrawalMicroseconds: limitsConfig.oldEnoughForWithdrawal,
+})
+
+export const generateFeeConstants = (feesConfig = yamlConfig.fees): FeeConstants => ({
+  depositFeeRate: feesConfig.deposit,
+  withdrawFeeFlat: feesConfig.withdraw,
+})
+
+export const selectUserLimits = ({
```

Comment:
I think is better to use `getUserLimits` like `getRateLimits`, `getUserWalletConfig` ...

Reply:
Agreed, I think I was trying to use different names based on whether I was passing arguments to select values, or whether the method was doing a straight mapping. Thinking back now, that probably doesn't make a whole lot of sense :sweat_smile: